packages/fxa-content-server/app/tests/spec/lib/app-start.js

```diff
@@ -139,6 +148,12 @@ describe('lib/app-start', () => {
     assert.isDefined(appStart._getUniqueUserId());
   });
 
+  it('initializeSurveyTargeter creates targeter', () => {
+    appStart.initializeSurveyTargeter();
+    assert.isDefined(appStart._surveyTargeter);
```

Comment:
We could check to see if the properties in the targeter instance are those that were passed into the constructor.